recipe/meta.yaml

```diff
@@ -22,11 +22,10 @@ requirements:
     - python
     - pip
   run:
-    - python
+    - python >=3.6
     - numpy
     - xmltodict
     - beautifulsoup4
-    - matplotlib-base
     - requests
```

Comment:
When I checked the new repo, I'm only seeing requests imported in the two files. Can all of these other dependencies be removed?

Reply:
The conda meta.yaml in the main repo still has all of these dependencies: https://github.com/williamgilpin/pypdb/blob/master/devtools/conda-recipe/meta.yaml but it isn't clear to me why.

Reply:
See that requests is all that is imported here:

https://github.com/williamgilpin/pypdb/blob/master/pypdb/pypdb.py#L24

Reply:
It looks like a lot of code has been commented on. Probably due to the soon-to-be-coming refactoring. I guess we should remove those deps for the moment.

Reply:
beautifulsoup4 and xmltodict are still required in 1.310 (check the tag on github).